samza-core/src/main/java/org/apache/samza/coordinator/stream/messages/SetExecutionContainerIdMapping.java

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
Either needs a merge from latest master or this is an extra file that needs to be removed since it got renamed in master and the merge is keeping this as is.